tests/run/classof.scala

```diff
@@ -1,8 +1,11 @@
+class ValueClass(val i: Integer) extends AnyVal
 class SomeClass
 
 object Test {
   def main(args: Array[String]): Unit = {
     val cls: Predef.Class[SomeClass] = classOf[SomeClass]
+    val valCls: Predef.Class[ValueClass] = classOf[ValueClass]
+
```

Comment:
It would be good to add a test, that tests the error that was triggered by previous build.
That could be:

```scala
val iCls: Class[Int] = classOf[Int]
```